src/nvidia_rag/utils/health_models.py

```
@@ -0,0 +1,93 @@
+    message: str = Field(max_length=4096, pattern=r"[\s\S]*", default="Service is up.")
+    databases: list[DatabaseHealthInfo] = Field(default_factory=list)
+    object_storage: list[StorageHealthInfo] = Field(default_factory=list)
+    nim: list[NIMServiceHealthInfo] = Field(
+        default_factory=list
+    )  # NIM services (embeddings, LLM, etc.)
+
+
+class RAGHealthResponse(HealthResponseBase):
+    """Health response for RAG server with database, storage, and NIM services"""
+
+    pass
```

Ruff (0.14.5)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)